lib/rubocop/cop/layout/indentation_consistency.rb

```diff
@@ -5,14 +5,118 @@ module Cop
     module Layout
       # This cops checks for inconsistent indentation.
       #
-      # @example
+      # The difference between `rails` and `normal` is that the `rails` style
+      # prescribes that in classes and modules the `protected` and `private`
+      # modifier keywords shall be indented the same as public methods and that
+      # protected and private members shall be indented one step more than the
+      # modifiers. Other than that, both styles mean that entities on the same
+      # logical depth shall have the same indentation.
       #
+      # @example EnforcedStyle: normal (default)
+      #   # bad
       #   class A
       #     def test
       #       puts 'hello'
       #        puts 'world'
       #     end
       #   end
+      #
+      #   # bad
+      #   class A
+      #     def test
+      #       puts 'hello'
+      #       puts 'world'
+      #     end
+      #
+      #     protected
+      #
+      #       def foo
+      #       end
+      #
+      #     private
+      #
+      #       def bar
+      #       end
+      #   end
+      #
+      #   # good
+      #   class A
+      #     def test
+      #       puts 'hello'
+      #       puts 'world'
+      #     end
+      #   end
+      #
+      #   # good
+      #   class A
+      #     def test
+      #       puts 'hello'
+      #        puts 'world'
```

Comment:
Oops! This is a typo. I'm sorry about self code review was lack.
I'm going to open PR to fix this.